doc/OnlineDocs/getting_started/installation.rst

```diff
@@ -86,16 +90,30 @@ initialized.
 .. note::
    This can only be done via ``pip`` or from source.
 
-Via ``pip``:
+Installation via ``pip`` or from source is done the same way - using environment
+variables. On Linux/MacOS:
 
 ::
 
-   pip install pyomo --global-option="--with-cython"
+   export PYOMO_SETUP_ARGS=--with-cython
+   pip install pyomo
+
+On Windows:
+
+::
+
+   # Via command prompt
+   set PYOMO_SETUP_ARGS=--with-cython
+   # Via powershell
+   $env:PYOMO_SETUP_ARGS="--with-cython"
```

Comment:
Does it matter that you have he call for `pip install pyomo` in Linux, but not here?

Reply:
Whoops. That was a mistake. Fixed!